backend/repopulate_db.py

```diff
@@ -42,7 +42,8 @@
     tree2 = {
         "id": 2,
         "data": {
-            "name": "Root",
+            "name": "Wisdom",
+            "url": "https://cwsl.ca/wiki/doku.php?id=philosophy_of_wisdom_socrates_and_plato#psychotechnologies_metacognition_and_second_order_thinkingimplications_for_the_machinery_of_meaning-making",
```

Comment:
All these hard-coded URL's are eventually going to be the job of the professors, right?

Reply:
correct; this is just for the first phase when we don't have professors entering data